### PR TITLE
 #167867889 Connect database with the application

### PR DESCRIPTION
**Pull request by nziokaivy**
#### What does this PR do?
Connects application to the database

#### Description of Task to be completed?
- Installed pg dependency.
- Configured database options in the config file
- Configured database URL in the environment.
- Create empty tables: trip, users and booking

#### How should this be manually tested?
Clone this repository and run `npm install && run test`

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
 #167867889

#### Screenshots (if appropriate)
Screenshot from 2019-08-13 19-27-50

### CODE REVIEW

File: UI/js/index.js [hunk 1]
```diff
@@ -1,3 +1,4 @@
+/* eslint-disable no-unused-vars */
 // functionality for tabs in profile page
 function openTab(evt, tabName) {
 	let i; let tabcontent; let
```
Comment (oesukam) [inline UI/js/index.js]: What is the last let used for?

File: server/api/config/config.js [hunk 2]
```diff
@@ -1 +1,16 @@
...
+	db: process.env.DATABASE_URL,
+};
+const test = {
+	db: process.env.DATABASE_TEST_URL,
+};
+const config = {
+	dev,
+	test,
+};
+module.exports = config[env];
```
Comment (oesukam) [inline server/api/config/config.js]: Why can't you use `ES6` with `export` or `export default`?

**Review by oesukam — CHANGES_REQUESTED** [review]

File: UI/js/index.js (hunk 1 above)
Comment (nziokaivy) [reply UI/js/index.js]: The variable was down below.
I have now edited it.

File: server/api/config/config.js (hunk 2 above)
Comment (nziokaivy) [reply server/api/config/config.js]: I have changed it.

**Review by oesukam — APPROVED** [review]

**Pull request merged**